client/elife-theme/src/elements/Button.js

```diff
@@ -0,0 +1,46 @@
+import { css } from 'styled-components'
+import { th, darken } from '@pubsweet/ui-toolkit'
+
+const extraSmall = css`
+  font-size: 11px;
+  line-height: calc(${th('gridUnit')} * 2);
+  padding: ${th('gridUnit')};
+`
+
+const small = css`
+  padding: ${th('gridUnit')};
+`
+
+const primary = css`
+  border-color: ${th('colorPrimary')};
+
+  &:focus,
+  &:hover {
+    border-color: ${darken('colorPrimary', 0.3)};
+  }
+
+  &:active {
+    border-color: ${darken('colorPrimary', 0.5)};
+  }
+
+  &[disabled] {
+    &:focus,
+    &:hover,
+    &:active {
+      border-color: ${th('colorPrimary')};
+    }
+  }
+`
+
+// TODO this should be updated when pubsweet changes gridUnit to 8px
```

Comment:
why? to what?

Reply:
Again, check the discussion me & tam had.